hooks/test_conditions.js

```diff
@@ -47,5 +47,18 @@ module.exports = async function () {
         });
       }
     }
+
+    if (suite.title === 'nonDBGap Project Usersync @requires-fence @requires-indexd') {
+      const dbGaPConfig = bash.runCommand('gen3 secrets decode fence-config fence-config.yaml | yq -r \'.dbGaP[] | ."info".study_to_resource_namespaces | ."PROJECT-12345"\'');
```

Comment:
still unsure if this is the right way of checking when to run the test in test conditions. Still have to talk to Binam about what can be best way of checking